common/presenters/table/object-to-table-head.test.js

```diff
@@ -1,28 +1,84 @@
 const i18n = require('../../../config/i18n')
 
-const schemaExample = [
-  {
-    head: 'move_type::label',
-    row: 'prison_transfer_reason',
-  },
-  {
-    head: 'fields::age',
-    row: 'age',
-  },
-]
 const objectToTableHead = require('./object-to-table-head')
 
 describe('table head presenter', function() {
-  it('returns the heading specified', function() {
+  beforeEach(function() {
     sinon.stub(i18n, 't').returnsArg(0)
-    const output = schemaExample.map(objectToTableHead)
-    expect(output).to.deep.equal([
+  })
+  context('with html', function() {
```

Comment:
I think you're missing two scenarios from this test:

`with both html and text`

and 

`with neither html or text`

Reply:
Again, I will add the tests, but how is this a realistic scenario?

Reply:
Fine, let's leave this scenario for now. My point was more around the tests not describing what the current intended behaviour is in those scenarios.

It doesn't show that if you pass neither or both what actually happens. Which means that if the behaviour were to change in the future you wouldn't see the side effects based on the original intended behaviour. 

I think the tests can be more descriptive about what this presenter does in certain scenarios rather than just creating one high level test that ticks the code coverage box by running over each line at least once.